win32_deps_build.sh

```diff
@@ -24,7 +24,7 @@ curlSrcDir="${depsSrcDir}/curl"
 curlDir="${depsToolsetDir}/curl"
 
 # For now, we'll keep the version number within the file path when not using git.
-boostUrl="https://dl.bintray.com/boostorg/release/1.73.0/source/boost_1_73_0.tar.gz"
+boostUrl="https://boostorg.jfrog.io/artifactory/main/release/1.73.0/source/boost_1_73_0.tar.gz"
```

Comment:
since you are at here, could you use boost 1.75 to be in sync with `CMakeLists.txt` ?

Reply:
Sure, I think that would also allow us to drop some of the boost patches that we apply since those were merged upstream in the meantime.